pages/treeherder.py

```diff
@@ -7,15 +7,20 @@
 from selenium.webdriver.support.ui import WebDriverWait as Wait
 
 import expected
+import random
```

Comment:
Standard library imports should be at the top, separated from other imports with a single blank line.